src/com/company/M3/Task04.java

```diff
@@ -0,0 +1,17 @@
+package com.company.M3;
+import java.util.Scanner;
+
+public class Task04 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String name = scanner.nextLine();
+        String surname = scanner.nextLine();
+        int age = scanner.nextInt();
+        createForms(name, surname, age);
+    }
+
+    public static void createForms(String name, String surname, int age) {
+        System.out.println(name + " " + surname + " " + (age * 10) + " ");
```

Comment:
let's introduce a new variable for `age * 10` this will help avoid code duplication in line 14 and 15